Makefile

```diff
@@ -62,6 +62,12 @@ install-dev:
 install-build:
 	python3 -m pip install -r requirements/build.txt
 
+.PHONY: install-code
+install-code:
+	python3 -m pip install -r requirements/extra-code.txt
+	scripts/install_treesitter.sh
```

Comment:
Hello
I remove that script in favor of dynamically building the treesitter library at runtime

However I have a small dilemma, treesitter is loading a shared library to be able to parse code, and there are only dirty method to actually unload libraries in python

Now, what I was thinking of is to build new libraries when new languages to parse show up, updating the main one as well for future use, and cleaning the extra built libraries whenever we leave the script/program that uses the parser. This works, but it means that there is a increase in memory usage for all new language that is greater than what it should actually be, which might be a problem if you spin up multiple API endpoint for load balancing or whatnot

We can partially fix it by setting a list of supported languages, and just build that the first time the module is imported. Or by using all the languages (even if the probability of people parsing things like Motorola 68000 Assembly is quite low)

In the end, to me anything is fine since I don't have infrastructure running for clients, so it's up to you

Reply:
Went for the install on first load in the current commits